tests/test_extract_inventory_numbers.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+import re
+from extract_inventory_numbers import _compile_regex
+
+
+class TestRegEx(unittest.TestCase):
+
+    def test_regex(self):
+        # test cases comprised of tuples
+        # where the first item is the input string
+        # and the second item is the expected output derived from FTVA specs
+        test_cases = (
```

Comment:
Please add test cases for `T` and `DVD`.  Yes, they should work the same as all the others prefixes, but let the tests prove that.